public/index.html

```diff
@@ -66,6 +66,7 @@
           },
           maskedface: null,
           loading: '',
+          status: '',
```

Comment:
Why did you add a status attribute here? 

You added the `status` attribute but did not use it at all.

Reply:
my bad, had it there when I was testing some stuff, will remove it